### PR TITLE
Provisioning scripts for vagrant

### PR DESCRIPTION
**Pull request by gravelweb**
Tested this from clean, and reruning from `vagrant ssh` via `cd /vagrant && bash install-ggtrackerstack.sh`.

Depends on two other pull requests:
https://github.com/nickelsen/ggtracker/pull/2
https://github.com/nickelsen/esdb/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ Project to run the whole ggtracker stack in vagrant
...
+ * Run `vagrant up` to create the virtual box. This will download all required packages and go through the installation process for ggtracker and ESDB (~15 minutes; go make yourself a sandwich)
  * Run `vagrant ssh` to ssh into the vagrant box
- * Find code in /vagrant
- * Run the installation and updating steps of the components (ESDB and ggtracker) ([will be automated soon](https://github.com/gravelweb/ggtrackerstack/commit/e3bdbb30d9384d37e8b96692af81cefd5f8a87d2))
-   * ESDB: https://github.com/dsjoerg/esdb#installation-and-setup
-   * ggtracker: https://github.com/dsjoerg/ggtracker#basic-installation-and-updating
-     * Note: You need to run ggtracker webapp like this `ESDB_HOST=172.28.128.3:9292 foreman start` (change ip accordingly)
+ * Find code in /vagrant `cd /vagrant`
+ * Start the application (requires 2 ssh sessions, or run them as background jobs)
+   * ESDB: `(cd esdb && ESDB_HOST=172.28.128.3:9292 foreman start)` (change IP accordingly)
```
Comment (nickelsen) [inline README.md]: Isn't this the other way around - ESDB_HOST is a parameter for ggtracker's `foreman start`?

**Comment by nickelsen** [comment]
This is awesome - I did this manually so many times while testing the stack - will save soo much time!

Once the esdb dependency is merged, we'll pull this!

File: README.md (hunk 1 above)
Comment (dsjoerg) [inline README.md]: @nickelsen you are correct, it's ggtracker that needs to know the ESDB_HOST

**Comment by dsjoerg** [comment]
@nickelsen when you say "esdb dependency" are you referring this pull request that has been merged already?  https://github.com/dsjoerg/esdb/pull/4

**Comment by dsjoerg** [comment]
This PR is still waiting on https://github.com/nickelsen/esdb/pull/1 which has been requested to be re-submitted against dsjoerg/esdb

File: README.md (hunk 1 above)
Comment (gravelweb) [inline README.md]: D'oh. Good catch. Fixed now.

**Comment by gravelweb** [comment]
Now waiting on [dsjoerg/esdb#5](https://github.com/dsjoerg/esdb/pull/5)

**Pull request merged**